cfn-templates/sso-permission-sets.cf.yaml

```diff
@@ -146,6 +146,9 @@ Resources:
           - Effect: Allow
             Action: apigateway:POST
             Resource: !Sub arn:${AWS::Partition}:apigateway:*::/restapis/*/deployments
+          - Effect: Allow
+            Action: secretsmanager:GetSecretValue
+            Resource: "*"
```

Comment:
```suggestion
```
I would delete this statement and not attach to the developer role. 
Developers can use the secret read access instead.